stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/mediagallerypreview/MediaGalleryPreviewOption.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.compose.state.mediagallerypreview
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+
```

Comment:
Replica of `ImagePreviewOption` with the addition of `isEnabled` for an easier time of disabling features that don't work offline or are too complicated to work offline.